.github/dependabot.yml

```
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  # Enable version updates for Docker
+  - package-ecosystem: "docker"
+    # Look for a `Dockerfile` in the `build` directory
+    directory: "/build/dockerfiles"
+    # Check for updates once a week
```

In accordance with the provided interval:

```suggestion
    # Check for updates every weekday
```

---

or better, check weekly instead :)